README.md

```diff
@@ -69,6 +69,8 @@ Don't forget to pass a `ref` to the component.
 
 You can change the initial options passing props to the component.
 
+**autoResizeHole** {bool}: Enable autoresizing holes when the content under the hole change its size. Defaults to `false`
```

Comment:
Well because I am not really sure how big is the overhead when you listen for events. Maybe for really big apps where you will have a lots of events it can be a problem - not sure, I did not test it.